Core/Core/DocViewer/DocViewerSession.swift

```diff
@@ -65,8 +54,6 @@ class DocViewerSession: NSObject, URLSessionTaskDelegate {
             } else {
                 self?.loadMetadata(sessionURL: nil)
             }
-            self?.noFollowSession?.invalidateAndCancel()
```

Comment:
we don't need this anymore?

Reply:
We used to create a `URLSession` per `DocViewerSession`, and to keep it from leaking memory we'd invalidate it. That made it hard to mock. Now instead we have a single static `NoFollowRedirect.session` for that, and invalidating it would break subsequent doc viewing sessions.